README.md

```diff
@@ -3,6 +3,8 @@ API Server for CogniCity
 
 [![Build Status](https://travis-ci.org/urbanriskmap/cognicity-server.svg?branch=master)](https://travis-ci.org/urbanriskmap/cognicity-server)
 
+[![Coverage Status](https://coveralls.io/repos/github/urbanriskmap/cognicity-server/badge.svg?branch=server-object-refactor)](https://coveralls.io/github/urbanriskmap/cognicity-server?branch=server-object-refactor)
+
```

Comment:
Should we think about including mock data in the db so that we can check actual replies instead of 200s? If you don't want to overwrite developer's dbs we can think about docker/ making a different db test_cognicity. Would be really nice to check that the /floods endpoint gives you the right info that's supposed to be in the db. Could also do a post and then check that data is on db.